src/fields/location/index.ts

```diff
@@ -0,0 +1,17 @@
+import type { Field } from 'payload'
+
+export const locationField: Field = {
+  type: 'row',
+  fields: [
```

Comment:
In payload there is point type. I am not sure which one is more convenient for our case: https://payloadcms.com/docs/fields/point

Reply:
makes sense